package.json

```diff
@@ -65,9 +67,11 @@
     "express-jwt": "latest",
     "get-port": "latest",
     "jest": "latest",
+    "jest-puppeteer": "latest",
     "jsdom": "latest",
     "lodash.get": "latest",
     "nuxt-edge": "latest",
+    "oidc-provider": "^6.29.9",
```

Comment:
Why not "latest"?

Reply:
I'm finding several other versioned packages. How should I move forward with this newly introduced dependency?

Reply:
Yeah looks like we've been inconsistent here. Good for now.